stable/airflow/templates/deployments-scheduler.yaml

```diff
@@ -280,6 +292,12 @@ spec:
           configMap:
             name: {{ .configMap }}
         {{- end }}
+        {{- if .Values.airflow.extraStartupScripts }}
+        - name: extra-startup-scripts
+          configMap:
+            name: {{ .Values.airflow.extraStartupScripts }}
+            defaultMode: 0740
```

Comment:
I got permission denied, I think this needs to be more permissive like 755:
```
[2020-02-26 23:48:53,468] {{__init__.py:51}} INFO - Using executor CeleryExecutor
1 of 1 variables successfully updated.
running extra startup scripts
bash: /usr/local/extra-startup-scripts/run.sh: Permission denied
```

Reply:
A `755` would not change anything since it would mean `u=rwx,g=rw,o=rw`. Normally `u=rwx,g=r,o=` (i.e. `740`) should've sufficed. 

I'll change it to `770` then, maybe the files are executed via the group and not the user OpenShift style...

Reply:
Done.

Reply:
OK, I pulled your changes and 770 does not work either:
```
[2020-02-28 15:21:03,242] {{__init__.py:51}} INFO - Using executor CeleryExecutor
4 of 4 variables successfully updated.
running extra startup scripts
bash: /usr/local/extra-startup-scripts/run.sh: Permission denied
```

Reply:
@NBardelot did you try it yourself? If you don't get the same permissions error, I'd be curious why... I would think you'd get the same error though.

Reply:
I've taken a more thorough look at it and...

1. you were absolutely right about 755 (which is `u=rwx,g=rx,o=rx` and *not* what I said... stupid me)

2. `o=rx` is needed because the airflow user in not included in the `root` group as it should be (it is a good practice for Docker/Kubernetes and especially useful when running OpenShift)

I'll fix this right away. Sorry for that.

Reply:
hey @NBardelot no worries! I have been running this branch out, so I knew there was something up. I appreciate your efforts. 

Now, it would be good to get it merged in :)

Reply:
I have force-pushed the fix so the pull request is ready now. If you can confirm that this version of the commit works for you it'd be great.

Reply:
Confirmed (I already had the change exactly as you have it local).